src/main/java/gibson/task/Parser.java

```diff
@@ -0,0 +1,191 @@
+            }
+            // EVENTS
+        } else if (Pattern.matches("(event .+ /at .+)|(event .+ /at( )?)", input)) {
+            String taskString = Parser.substringAfterToken(input, "event");
+            String[] stringArray = Parser.substringBeforeAfterToken(taskString, "/at");
+            try {
+                Event event = new Event(stringArray[0], stringArray[1]);
+                taskList.add(event);
+                storage.save(taskList.toString());
+                return "Got it. I've added this task:\n"
+                        + event
+                        + "\nNow you have " + taskList.size() + " task(s) in the list.";
+            } catch (IllegalArgumentException e) {
+                return e.getMessage();
+            }
+            // REMOVE
+        } else if (Pattern.matches("delete [0-9]+", input)) {
+            int number = Parser.getTrailingInt(input);
+            int index = number - 1;
+            try {
+                Task t = taskList.remove(index);
+                storage.save(taskList.toString());
+                return "Noted. I've removed this task:\n"
+                        + t.toString()
+                        + "\nNow you have " + taskList.size() + " task(s) in the list.";
+            } catch (IndexOutOfBoundsException e) {
+                return "There is not task numbered as " + number + ".";
+            }
+            // FIND
+        } else if (Pattern.matches("(find .+)|(find( )?)", input)) {
+            String searchQuery = Parser.substringAfterToken(input, "find");
+            TaskList result = taskList.find(searchQuery);
+            if (!result.isEmpty()) {
+                return "Here are your results:\n" + result.toString();
+            } else {
+                return "Unable to find task with \"" + searchQuery + "\".";
+            }
+            // NOT RECOGNIZED
+        } else {
+            return "I'm sorry. I do not know what that means.";
+        }
+    }
+
+    /**
+     * Returns trailing integers from a given string.
+     * @param string the string to get trailing integers from
+     * @return the trailing integers
+     */
+    public static int getTrailingInt(String string) {
+        Pattern pattern = Pattern.compile("[^0-9]+([0-9]+)$");
+        Matcher matcher = pattern.matcher(string);
+        if (matcher.find()) {
+            return Integer.parseInt(matcher.group(1));
+        } else {
+            throw new IllegalArgumentException("String is invalid");
+        }
+    }
+
+    /**
+     * Returns substring after the first instance of the token string.
+     * @param string the string to be substring-ed
+     * @param token the string that marks where to substring-ed
+     * @return substring after the token string
+     * @throws IllegalArgumentException if token is not found in the string
+     */
+    public static String substringAfterToken(String string, String token) {
+        Pattern pattern = Pattern.compile(token);
```

Comment:
Considering it is a method, do you think it would be better to perhaps name it using a verb as per the given [coding standards](https://se-education.org/guides/conventions/java/intermediate.html)? The same goes for other such methods as well.